mcc/got/cmd/delete.go

```diff
@@ -0,0 +1,38 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// deleteCmd represents the delete command
+var deleteCmd = &cobra.Command{
+	Use:   "delete",
+	Short: "A brief description of your command",
+	Long: `A longer description that spans multiple lines and likely contains examples
+and usage of using your command. For example:
```

Comment:
Not sure if I understand this correctly, but this seems like a generated text from cobra, right? Maybe we could change it to something more specific/descriptive.